Resources/views/Calendar/view.html.twig

```diff
@@ -119,7 +119,15 @@
                             <hr/>
                         </div>
                         {% endif %}
-                        <div><label class="bold">{{'global.stop_point_code'|trans({}, 'messages')}}</label>: {{ calendar.schedules.stop_point.codes|externalCode }}</div>
+                        <div>
+                            <dl>
+                                <dt>{{'global.stop_point_code'|trans({}, 'messages')}}</dt>
+                                {% for code in calendar.schedules.stop_point.codes %}
+                                {% set value = code.type == 'external_code' ? code.value|slice(3) : code.value %}
```

Comment:
For me in the entity object we just need to have getter or setter not logical code. Here the goal of initStopPointCode in the StopPointManager service is to initialize all codes for the display view.
I think it's better to keep real data returned by navitia (with 3 digits) in the entity object if we need it in other place.